_posts/2021-04-07-#21061.md

```diff
@@ -7,19 +7,80 @@ authors: [amitiuttarwar]
 components: ["p2p"]
 host: glozow
 status: upcoming
-commit:
+commit: 038f751
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
+
+* Hiding links between wallet addresses and IP addresses is a key part of
+  Bitcoin privacy. Many techniques exist to help users obfuscate their IP
+address when submitting their own transactions, and various P2P changes have
+been proposed with the goal of hiding transaction origins.
+
+* However, _rebroadcast_ behavior can also leak information. If a node
```

Comment:
"goal of hiding transaction origins... However" 

since rebroadcast could also potentially leak tx origin, I think this combo is slightly confusing. I think you mean however to < initial tx broadcast >.

Reply:
"Beyond initial broadcast" ?